gocourse08/sensor/breath.go

```diff
@@ -0,0 +1,9 @@
+package sensor
+
+import "math/rand/v2"
+
+type BreathSensor struct{}
```

Comment:
```suggestion
type Breath struct{}
```

щоб уникнути повторюваності `sensor.BreathSensor`